modules/ROOT/pages/runtime-fabric-setup.adoc

```diff
@@ -22,12 +22,12 @@ Anypoint Visualizer relies on a header injection to display connections between
 Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to a potential performance impact in Mule 4, header injection is disabled by default.
 For Runtime Fabric, you must enable the header injection at the app level. You can enable the header injection in the *Properties* tab for an app:
```

Comment:
You know, now that we have the changed Step 1, given the previous sentence, I don't think we need the second. What do you think?
```suggestion
For Runtime Fabric, you must enable the header injection at the app level:
```